BREAKING_CHANGES.md

```diff
@@ -1,3 +1,6 @@
+## From 28.x to 29
+
+- **getOpenGLRenderWindow**: `getOpenGLRenderWindow` has been renamed to `getApiSpecificRenderWindow` in `vtkFullScreenRenderWindow`, `vtkGenericRenderWindow` and `vtkViewProxy` to support WebGL and WebGPU backend.
```

Comment:
Will you mark the commit as a breaking change?

Also, we have some other breaking changes that should go into v29. I've created a v29 milestone to track those, so that we can either merge them all at once or update the beta branch.

Reply:
Oops, I forgot to flag it as a breaking change, I'll do that.

I've no rush for that commit to be merged, so it can be merged whenever